packages/lesswrong/components/Layout.tsx

```diff
@@ -172,6 +179,12 @@ const styles = (theme: ThemeType): JssStyles => ({
       fontWeight: 600,
       marginTop: 20,
       marginBottom: 10,
+      textShadow: `
+        0 0 15px ${theme.palette.background.pageActiveAreaBackground}, 
+        0 0 15px ${theme.palette.background.pageActiveAreaBackground}, 
+        0 0 15px ${theme.palette.background.pageActiveAreaBackground}, 
+        0 0 15px ${theme.palette.background.pageActiveAreaBackground}
+      `,
```

Comment:
The other css classes being changed are all only used behind an `isLW`, but I don't know about `& h2` (and `& h3` below).  Not 100% sure what this is doing so might be worth forum-gating (or just behind `isFriendlyUI` if you're also pretty sure it's safe for AlignmentForum).

Reply:
these are h2s that are underneat "bannerText", which is underneath is LW, and only show up in a context where I'm pretty sure anyone else would also want the glow effect if they were choosing to render a banner. (to ensure readability)